report/renderer/crc-details-renderer.js

```diff
@@ -139,26 +137,24 @@ class CriticalRequestChainRenderer {
   /**
    * Recursively builds a tree from segments.
    * @param {DOM} dom
-   * @param {DocumentFragment} tmpl
```

Comment:
`tmpl` and `details` are never actually used anywhere, they just get passed through the recursive calls.